pkg/registry/apis/ofrep/register.go

```diff
@@ -225,8 +225,8 @@ func (b *APIBuilder) GetAPIRoutes(gv schema.GroupVersion) *builder.APIRoutes {
 
 func (b *APIBuilder) oneFlagHandler(w http.ResponseWriter, r *http.Request) {
 	if !b.validateNamespace(r) {
-		b.logger.Error("stackId in evaluation context does not match requested namespace")
-		http.Error(w, "stackId in evaluation context does not match requested namespace", http.StatusUnauthorized)
+		b.logger.Error("evaluation context namespace does not match namespace in request")
```

Comment:
I think we should make an error with this string since we keep repeating it 😉

Also, how about something like:

```suggestion
		b.logger.Error("rejecting request with namespace mismatch")
```